src/main/java/net/sf/jabref/gui/undo/UndoableChangeType.java

```diff
@@ -35,12 +36,19 @@ public UndoableChangeType(BibEntry entry, String oldType, String newType) {
 
     @Override
     public String getUndoPresentationName() {
-        return "Undo: change type";
+        return Localization.lang("Undo") + ": " + getPresentationName();
```

Comment:
Yes, that should indeed be possible. Didn't think of it. Earlier only some had a `getPresentationName` but based on the structure of the code, it made more sense to only change any significant text in one position.